src/openms_gui/source/VISUAL/SpectrumWidget.cpp

```diff
@@ -224,6 +224,7 @@ namespace OpenMS
   void SpectrumWidget::saveAsImage()
   {
     QString file_name = QFileDialog::getSaveFileName(this, "Save File", "", "Images (*.bmp *.png *.jpg *.gif)");
+    this->setStyleSheet("background: white");
```

Comment:
maybe store old stylesheet? and reset below?

Reply:
Suggestion: RAIICleanup for restoring old stylesheet